test/onnx/internal/test_diagnostics.py

```diff
@@ -522,6 +536,20 @@ def expensive_formatting_function() -> str:
                 "expensive_formatting_function should only be evaluated once after being wrapped under LazyString",
             )
 
+    def test_diagnostic_log_emit_correctly_formatted_string(self):
```

Comment:
Duplicated

Reply:
Intended duplication to cover both base diagnostic infra and subclassed infra for dynamo exporter. No intention to improve test infra in this PR to remove duplication though.